### PR TITLE
Windiris fixes

### LINKED ISSUE #95: docstring: windiris source
child of #73

### PR DESCRIPTION
**Pull request by achaikou**
closes #95

### CODE REVIEW

File: camille/source/windiris.py [hunk 1]
```diff
@@ -47,33 +51,87 @@ def _sqlite(start_date,
...
+        Path to windiris data root directory
+    tzinfo : datetime.tzinfo
+        Timezone format for returned timeseries
+    Returns
+    -------
+    function (str, datetime.datetime, datetime.datetime, str, str, str)
+        Function for reading windiris time series from db file
+
+        installation : str
+            Installation directory inside root, part of the file name
```
Comment (sveinung-r) [inline camille/source/windiris.py]: Installation is a name specifying which windmill to collect data from. The fact that it is also a folder and table name is not something the user of this function should be concerned with.

File: camille/source/windiris.py [hunk 2]
```diff
@@ -47,33 +51,87 @@ def _sqlite(start_date,
...
+
+        Returns
+        -------
+        pandas.DataFrame
+           Result dataframe
+
+    Examples
+    --------
+
+    Read year data from table 'wind' located under <root>/wind/wind_rtd.db
```
Comment (sveinung-r) [inline camille/source/windiris.py]: Same as above. The user is reading data from an installation for a given time interval. How this data is stored is irrelevant.

**Comment by achaikou** [comment]
Done. As well noticed I missed one timezone check

File: camille/source/windiris.py [hunk 3]
```diff
@@ -113,9 +113,6 @@ def windiris_internal(installation,
                           distance=None,
                           status=None):
 
-        if not os.path.isdir(root):
```
Comment (sveinung-r) [inline camille/source/windiris.py]: Why is this check removed?

File: camille/source/windiris.py [hunk 4]
```diff
@@ -75,8 +75,7 @@ def windiris(root, tzinfo=pytz.utc):
         Function for reading windiris time series from db file
 
         installation : str
-            Installation directory inside root, part of the file name
-            and sql table name at the same time
+            Name of the windmill to collect data from
```
Comment (sveinung-r) [inline camille/source/windiris.py]: Not entirely sure installation necessarily corresponds to a single windmill.

File: camille/source/windiris.py (hunk 3 above)
Comment (achaikou) [reply camille/source/windiris.py]: It's actually moved up to windiris function itself (as in bazefetcher). I accidentally left both during rebasing

File: camille/source/windiris.py (hunk 4 above)
Comment (achaikou) [reply camille/source/windiris.py]: Options?
Do you think just "Installation name" would be fine?

Comment (sveinung-r) [reply camille/source/windiris.py]: "Name of the installation to collect data from" would be fine.

File: camille/source/windiris.py (hunk 3 above)
Comment (sveinung-r) [reply camille/source/windiris.py]: Aha, I see it now. Thanks for the clarification.

**Pull request merged**